### ISSUE TITLE
[Helm] Incorrect indentation when passing multiple mountOptions to the storageclass

### ISSUE THREAD

**Issue by ewencodes** [kubernetes-csi/csi-driver-nfs#698]
**What happened**:
Helm template failed with custom values when creating the storageclass.

**What you expected to happen**:
Create the storageclass and place correct mount options.

**How to reproduce it**:
Add csi-driver-nfs as repository :
```sh
helm repo add csi-driver-nfs https://raw.githubusercontent.com/kubernetes-csi/csi-driver-nfs/master/charts
```

Create a values file like below :
```yaml
storageClass:
  create: true
  name: nfs-csi
  parameters:
    server: ***
    share: /srv/data-disk
    mountPermissions: "0777"
  reclaimPolicy: Delete
  volumeBindingMode: Immediate
  mountOptions: 
    - hard
    - nfsvers=3
```

And launch the `template` helm command :
```sh
helm template csi-driver-nfs/csi-driver-nfs -f values.yaml
```

This result as this storageclass that is not valid due to lack of white space (or tabs) before `- nfsvers=3` :
```yaml
install.go:218: [debug] Original chart version: ""
install.go:235: [debug] CHART PATH: /..../helm/repository/csi-driver-nfs-v4.7.0.tgz

---
# Source: csi-driver-nfs/templates/storageclass.yaml

---
apiVersion: storage.k8s.io/v1
kind: StorageClass
metadata:
  name: nfs-csi
  labels:
    app.kubernetes.io/instance: "release-name"
    app.kubernetes.io/managed-by: "Helm"
    app.kubernetes.io/name: "csi-driver-nfs"
    app.kubernetes.io/version: "v4.7.0"
    helm.sh/chart: "csi-driver-nfs-v4.7.0"
provisioner: nfs.csi.k8s.io
parameters:
  mountPermissions: "0777"
  server: ***
  share: /srv/data-disk
reclaimPolicy: Delete
volumeBindingMode: Immediate
mountOptions:
  - hard
- nfsvers=3

---
# Source: csi-driver-nfs/templates/csi-nfs-driverinfo.yaml
---
apiVersion: storage.k8s.io/v1
kind: CSIDriver
metadata:
  name: nfs.csi.k8s.io
spec:
  attachRequired: false
  volumeLifecycleModes:
    - Persistent
  fsGroupPolicy: File
```

**Anything else we need to know?**:
This fix is to replace the `{{ toYaml . }}` by `{{  toYaml . | nindent 2 }}`.

**Environment**:
- CSI Driver version: 4.7.0
- CSI Driver chart version : 4.7.0

**Comment by andyzhangx** [comment]
@ewencodes could you take a look at this fix PR? thx
https://github.com/kubernetes-csi/csi-driver-nfs/pull/701

**Comment by ewencodes** [comment]
Hi @andyzhangx thanks a lot for this PR. Seems good to me

**Comment by andyzhangx** [comment]
> Hi @andyzhangx thanks a lot for this PR. Seems good to me

@ewencodes PR is merged, the issue was fixed in v4.7.0 chart

**Comment by ewencodes** [comment]
Again thanks a lot for the fast support @andyzhangx 🙏
